tests/api/test_nanopubs.py

```diff
@@ -100,3 +99,45 @@ def test_attribute_view(self):
         self.assertEquals(json_content['label'], "Jane Doe")
         self.assertEquals(len(json_content['type']), 1)
         self.assertEquals(json_content['type'][0]['label'], 'Person')
+
+    def test_ontology_describe_view(self):
```

Comment:
This test doesn't really belong here, but I don't have a better place for it at the moment.